docs/kubectl.md

```diff
@@ -1037,3 +1038,57 @@ Usage:
 
 ```
 
+#### label
+Update the labels on a resource.
+
+If --overwrite is true, then existing labels can be overwritten, otherwise attempting to overwrite a label will result in an error.
+If --resource-version is specified, then updates will use this resource version, otherwise the existing resource-version will be used.
+
+Examples:
+  $ kubectl label pods foo unhealthy=true
+  <update a pod with the label 'unhealthy' and the value 'true'>
+
+  $ kubectl label --overwrite pods foo status=unhealthy
+  <update a pod with the label 'status' and the value 'unhealthy' overwritting an existing value>
+  
+  $ kubectl label pods foo status=unhealthy --resource-version=1
+  <update a pod with the label 'status' and the value 'unhealthy' if the resource is unchanged from version 1>
+
+Usage:
+```
+  kubectl label [--overwrite] <resource> <name> <key-1>=<val-1> ... <key-n>=<val-n> [--resource-version=<version>] [flags]
```

Comment:
We use `<id>` for all other commands. We should be consistent, and convert them all to `<name>` at some point (soon).